### PR TITLE
[6.5.0] Disable PyTest.testSmoke on macOS

### PR DESCRIPTION
**Pull request by bazel-io**
Due to https://github.com/bazelbuild/bazel/issues/20660

RELNOTES: None
Commit https://github.com/bazelbuild/bazel/commit/d9dc4fde4c8d06498f39b54a81d4e7fa1e5b5b30

PiperOrigin-RevId: 593115090
Change-Id: Ifc2a282dbd6dd8a3abfec987398388e5844af91c

### CODE REVIEW

File: src/test/py/bazel/py_test.py [hunk 1]
```diff
@@ -46,6 +46,9 @@ def createSimpleFiles(self):
         ])
 
   def testSmoke(self):
+    if test_base.TestBase.IsDarwin():
```
Comment (meteorcloudy) [inline src/test/py/bazel/py_test.py:49]: Looks like `IsDarwin` doesn't exist in 6.x. Can you replace it with `sys.platform == 'darwin'`?

Comment (meteorcloudy) [reply src/test/py/bazel/py_test.py:49]: You'll also need to `import sys` first.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: src/test/py/bazel/py_test.py (hunk 1 above)
Comment (iancha1992) [reply src/test/py/bazel/py_test.py]: Done.

cc: @meteorcloudy @Wyverald

Comment (meteorcloudy) [reply src/test/py/bazel/py_test.py]: Can you please also disable the failing `//scripts/docs:rewriter_test` from both presubmit.yml and postsubmit.yml for macOS to fix the presubmit?

Comment (iancha1992) [reply src/test/py/bazel/py_test.py]: @meteorcloudy here's the PR: https://github.com/bazelbuild/bazel/pull/20758

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**